charts/kube-state-metrics/templates/_helpers.tpl

```diff
@@ -68,6 +68,9 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- if .Values.customLabels }}
 {{ toYaml .Values.customLabels }}
 {{- end }}
+{{- if .Values.releaseLabel }}
```

Comment:
IMO the if condition around the label should be removed: the _release_ label in a must-have for Prom to pick-up kube-state-metrics target.

Reply:
But Kube-state-metrics can also be used without Kube-Prometheus-stack too. Therefore no must have.